app-pda/libusbmuxd/libusbmuxd-1.0.10.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A client library to multiplex connections from and to iOS devices"
+DESCRIPTION="libusbmuxd is a client library to multiplex connections from and \
+to iOS devices by connecting to a socket provided by a usbmuxd daemon"
```

Comment:
Unlike SUMMARY, DESCRIPTION needs a full stop.

BTW, if anyone thinks we should change DESCRIPTION a little please do not hesitate to make suggestions ;-)